data/mesongenbuildreq.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
```

Comment:
as you call the script as `mesongenbuildreq` directly, you definitively want a shebang `#/usr/bin/python3` or `/usr/bin/env python3` (and the script file is called mesongenbuildreq.py - so you have clear expectations about the renaming of the script into the right location to use it - probably  justifies some doc)

Otherwise, this will be treated as a shell script when not started as `python3 mesongenbuildreq`